sim/battle.js

```diff
@@ -1162,12 +1157,13 @@ class Battle extends Dex.ModdedDex {
 
 		let eventHandler = {callback, target, priority, order, subOrder};
 
-		let callbackName = `on${eventid}`;
 		if (!this.events) this.events = {};
```

Comment:
Honestly, this entire function is fucked up. :( I didn't write it, and I'm not sure how I would have implemented it if it were me, but not like this.

Reply:
It looks like it's exclusively used by tests. Feel free to refactor it for that use-case.

Reply:
Didn't notice it was test-only, just looked for anything with the word 'event' in it for this refactoring pass. Thanks for the heads up, probably not worth optimizing ;)